static_src/test/unit/stores/service_instance_store.spec.js

```diff
@@ -286,16 +286,35 @@ describe('ServiceInstanceStore', function() {
     });
   });
 
-  describe('on instance create error', function() {
-    it('should set create error to passed in error', function() {
-      var expectedError = { status: 500 };
+  describe('on SERVICE_INSTANCE_CREATE_ERROR', () => {
+    it('should set `createError` based on error received', () => {
+      const serverError = { code: 500 };
+      const argumentError = { response: { data: { error_code: 'CF-MessageParseError' } } };
+      const spaceError = { response: { data: { error_code: 'CF-ServiceInstanceInvalid' } } };
+      const serverErrorMsg =
+        'Error #500: please contact cloud.gov support for help troubleshooting this issue.';
```

Comment:
Yes I agree. Do we have any concept of the app environment baked into the build process or anything? I think we can have webpack set a global variable during the build process that gets injected into the app code. Then 'cloud.gov' could just be whatever platform the dashboard is running on.